### DISCUSSION TITLE
Model and File locations

### DISCUSSION THREAD

**Discussion by OnlineNewsVentures** [gitmylo/audio-webui#115] [Q&A]
Can I import a tortoise voice to use in TTS?
Similarly, how can I use an RVC voice pth model created outside of the app?

**Comment by gitmylo** [comment]
* Custom tortoise models are currently not supported (unless maybe you replaced the one loaded by coqui TTS)
* RVC models are stored in `data/models/rvc/name/name.pth` (note the extra `/name/`, this allows you to put the index file in the same folder)
